src/Pipedrive.net/Models/Response/Leads/AbstractLead.cs

```diff
@@ -20,28 +20,26 @@ public abstract class AbstractLead
         [JsonProperty("label_ids")]
         public List<Guid> LabelIds { get; set; }
 
-        public CurrencyAmount Value { get; set; }
-
-        [JsonProperty("expected_close_date")]
-        public DateTime? ExpectedCloseDate { get; set; }
-
-        public string Note { get; set; }
-
         [JsonProperty("person_id")]
-        public long PersonId { get; set; }
+        public long? PersonId { get; set; }
```

Comment:
Added null possible for `person_id` because, this value can be null. And we receive JsonSerialization exception on try set null in not-nullable `PersonId`

Exception:
```
Unhandled exception. Newtonsoft.Json.JsonSerializationException: Error converting value {null} to type 'System.Int64'. Path 'person_id'
```